eclipse-collections-forkjoin/pom.xml

```diff
@@ -63,7 +63,21 @@
         <plugins>
 
             <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.plugin.version}</version>
```

Comment:
This declaration and similar ones seem to be redundant since plugin already declared in parent POM.